src/bin/fuse-query.rs

```diff
@@ -2,14 +2,13 @@
 //
 // SPDX-License-Identifier: Apache-2.0.
 
-use log::info;
-
 use fuse_query::clusters::Cluster;
 use fuse_query::configs::Config;
 use fuse_query::metrics::MetricService;
 use fuse_query::rpcs::{HttpService, RpcService};
 use fuse_query::servers::MySQLHandler;
 use fuse_query::sessions::Session;
+use log::info;
```

Comment:
This is strange, why move here?

Reply:
For `fuse-query.rs `,  crate `fuse_query` and `log` are all thirdparty crates.

Reply:
Got, I see